gitops/snapshot.go

```diff
@@ -122,6 +128,26 @@ func SetSnapshotIntegrationStatusAsInvalid(snapshot *applicationapiv1alpha1.Snap
 	})
 }
 
+// SetSnapshotIntegrationStatusAsInProgress sets the HACBS integration status condition for the Snapshot to In Progress.
+func SetSnapshotIntegrationStatusAsInProgress(snapshot *applicationapiv1alpha1.Snapshot, message string) {
```

Comment:
Since we will be using these functions in a different manner from `SetSnapshotIntegrationStatusAsInvalid`, I believe that we will need to make these functions also patch the resource on the cluster, similar to [MarkSnapshotAsPassed](https://github.com/redhat-appstudio/integration-service/blob/main/gitops/snapshot.go#L84-L94).

Reply:
Updated. Thanks.